roottest/root/aclic/nolinkdep/CMakeLists.txt

```diff
@@ -1 +1,49 @@
-# ROOTTEST_ADD_TEST(link MACRO link.C MACROARG 1)
+ROOTTEST_COMPILE_MACRO(single.C
+                       FIXTURES_SETUP root-aclic-nolinkdep-single_C-fixture)
+
+ROOTTEST_COMPILE_MACRO(script1.C
+                       FIXTURES_SETUP root-aclic-nolinkdep-script1_C-fixture)
+
+ROOTTEST_COMPILE_MACRO(script2.C
+                       BUILDLIB single_C
+                       FIXTURES_REQUIRED root-aclic-nolinkdep-single_C-fixture
+                       FIXTURES_SETUP root-aclic-nolinkdep-script2_C-fixture)
+
+ROOTTEST_ADD_TEST(single
+                  MACRO single.C+
+                  FIXTURES_REQUIRED root-aclic-nolinkdep-single_C-fixture
+                  FIXTURES_SETUP root-aclic-nolinkdep-single-fixture)
+
+ROOTTEST_ADD_TEST(load1
+                  MACRO script1.C+
+                  FIXTURES_REQUIRED root-aclic-nolinkdep-script1_C-fixture
+                  FIXTURES_SETUP root-aclic-nolinkdep-load1-fixture)
+
+ROOTTEST_ADD_TEST(load2
+                  MACRO script2.C+
```

Comment:
I am unclear whether the intent was to run this script with or without `script1_C` being there ...